package.json

```diff
@@ -32,8 +32,8 @@
     "lib"
   ],
   "scripts": {
-    "test": "make test",
-    "postinstall": "cd node_modules/monk/node_modules/mongodb && npm uninstall bson && cd ../../../; cd node_modules/mongodb && npm uninstall bson"
+    "test": "mocha --harmony test/mongorito.test",
```

Comment:
I disagree the make file relies on a very specific path to mocha where as this will use node_modules load order to find and use the right version of mocha.